### PR TITLE
pikachu attack

### PR DESCRIPTION
**Pull request by Fredrick-Tam**
"Please review this, @casey-collab"

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2161)

### CODE REVIEW

File: english/pikachu/pokeball.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+I cant be in a pokeball.
+
+They dont exist!!
+
+Its time to get up!
+
+I must resolve this marshmallow dream.
+I must get revenge from pikachu as well but maybe later....
+
+[Wake Up](awake/awake.md)
```
Comment (casey-collab) [inline english/pikachu/pokeball.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/pikachu/pokeball.md (hunk 1 above)
Comment (Fredrick-Tam) [inline english/pikachu/pokeball.md]: "Please review this, @casey-collab"

**Comment by Fredrick-Tam** [comment]
"Please review this, @casey-collab"

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**